rust/private/repository_utils.bzl

```diff
@@ -526,8 +526,7 @@ def load_arbitrary_tool(ctx, tool_name, tool_subdirectories, version, iso_date,
     # View the indices mentioned in the docstring to find the tool_suburl for a given
     # tool.
     tool_suburl = produce_tool_suburl(tool_name, target_triple, version, iso_date)
-    static_rust = ctx.os.environ.get("STATIC_RUST_URL", "https://static.rust-lang.org")
-    urls = ["{}/dist/{}.tar.gz".format(static_rust, tool_suburl)]
+    urls = []
```

Comment:
This code used to allow adding a URL by setting an env var, and this PR removes that ability - how would you feel about making this look something like:
```py
urls = []
static_rust_url_from_env = ctx.os.environ.get("STATIC_RUST_URL")
if static_rust_url_from_env:
    urls.append("{}/dist/{}.tar.gz".format(static_rust_url_from_env, tool_suburl))
```

that way if the env var isn't set, it still behaves exactly as you've written, but if it is we still pay attention to it.

Reply:
Thats a good point. +1